landlab/grid/hex.py

```diff
@@ -691,6 +691,8 @@ def hexplot(self, data, data_label=None, color_map=None):
 
         try:
             self._hexplot_configured is True
+            if self._hexplot_pc.cmap != color_map:
```

Comment:
This try/except should be cleaned up. The bare except will catch *any* error and so should be more specific. Probably it should be `except AttributeError`? The newly added `if` statement should probably be in the try/except `else` block. In addition, since the `is True` part doesn't do anything, it should be dropped.

So maybe the following?
```python
        try:
              self._hexplot_configured
        except AttributeError:
            self._configure_hexplot(data, data_label, color_map)
        else:
            if self._hexplot_pc.cmap != color_map:
                self._configure_hexplot(data, data_label, color_map)
```